ELK/filebeat.yml

```
@@ -0,0 +1,18 @@
+filebeat.inputs:
+- type: log
+  enabled: true
+  paths:
+    - /app/logs/*.json
+  json.keys_under_root: true
+  json.overwrite_keys: true
+
+processors:
+  - add_fields:
```

yamllint

[warning] wrong indentation: expected 0 but found 2

(indentation)